src/overdub.jl

```diff
@@ -39,7 +39,9 @@ else
     specialize_method(method, metharg, methsp, world, force) = Core.Compiler.specialize_method(method, metharg, methsp, force)
 end
 
-
+function overdubbed_iterate(ctx, iterate)
+    (args...) -> overdub(ctx, iterate)
+end
```

Comment:
I think this needs to take (and be passed) additional args, e.g. 
```julia
function overdubbed_iterate(ctx, iterate, args...)
    (args...) -> overdub(ctx, iterate, args...)
end
```

Reply:
But doing that causes test failuse elsewhere, (VATupleCtx test) but it makes all the Nuke tests pass.